3main

```diff
@@ -1040,10 +1042,12 @@ then
 
   TI="3"
 
-  if [[ $___1050_or_1050ti == YES ]]
+  if [ $___1050_or_1050ti == "YES" ]
   then
     TI="2"
-    if [[ $NORMAL == YES ]]
+    # Line 165 we have NORMAL=YES if cards are 1060|1070|1080
```

Comment:
Line 165 we have NORMAL=YES if cards are 1060|1070|1080
I think the NORMAL=YES statement should be separated